cmd/broker/deprovisioning.go

```diff
@@ -97,7 +97,7 @@ func NewDeprovisioningProcessingQueue(ctx context.Context, workersAmount int, de
 		}
 	}
 
-	queue := process.NewQueue(deprovisionManager, logs)
+	queue := process.NewQueue(deprovisionManager, logs, "deprovisioning-queue", cfg.Broker.WorkerHealthCheckWarnInterval, cfg.Broker.WorkerHealthCheckInterval)
```

Comment:
nit: "privisioning-queue" -> "provisioning" - we don't need to name the queue with word "queue", we know it :)

Reply:
the same for provisioning and an update